packages/calcite-components/src/components/color-picker-hex-input/color-picker-hex-input.tsx

```diff
@@ -452,4 +468,14 @@ export class ColorPickerHexInput implements LoadableComponent {
 
     return Color(nudgedChannels);
   }
+
+  /* 
+      if alphaChannel is enabled, commit valid 4 digit hex codes.
+      if alphaChannel is disabled, commit valid 3 digit hex codes.
+    */
+  private validateShortHandValue = (value: string): boolean => {
```

Comment:
```suggestion
private validateShortHandValue = (value: string): boolean => {
  const  length = value?.length;
  const {alphaChannel } = this;
    return (
      (alphaChannel && length === 4) || (!alphaChannel && length === 3)
    );
  };
```

Reply:
Franco pointed out this [util](https://github.com/Esri/calcite-design-system/blob/dev/packages/calcite-components/src/components/color-picker/utils.ts#L34) I'm using this and getting rid of `validateShortHandValue`.